str_analysis/call_non_ref_pathogenic_motifs.py

```diff
@@ -953,6 +1025,29 @@ def process_locus(locus_id, args, normalize_to_coverage=NORMALIZE_TO_COVERAGE, m
 
     print(f"Final call: {final_call}")
 
+    # Process STRling results
+    if strling_genotype_df is not None:
+        strling_records = parse_strling_info_for_locus(strling_genotype_df, locus_chrom, locus_start_0based, locus_end)
+        for i in 0, 1:
+            record = {}
+            if i < len(strling_records):
+                record = strling_records[i]
+            elif i > 0:
+                break
```

Comment:
could this section be an if statement making sure the length of `strling_records` is only 1 (instead of this small loop)?

Reply:
I still need the loop to avoid duplicating the `locus_results_json.update({... ` code block (unless I move it to a function). I've rewritten the if statements to be easier to read.